### PR TITLE
Add open source design token validator

### PR DESCRIPTION
**Pull request by andrico1234**
Hi @klaufel ,

I just created a free online tool to validate design tokens. It checks against the rules of the Design Token Community Group specs

I think it'd be a great addition to your list :)

### CODE REVIEW

**Comment by klaufel** [comment]
Hi @andrico1234 !! it is an amazing tool! thanks for the contribution!!

File: README.md [hunk 1]
```diff
@@ -90,6 +90,7 @@ See design tools plugins [here](https://github.com/LisaDziuba/Awesome-Design-Too
 - [Design Tokens W3C Community Group](https://www.w3.org/community/design-tokens/) - View repository on GitHub [here](https://github.com/design-tokens/community-group)!
 - [DesignTokens.dev](https://www.designtokens.dev/) - Ship your design tokens without managing infrastructure.
 - [Diez](https://diez.org/) - Free & open-source developer toolkit for expressing visual styles that can be shared across codebases, native platforms, and teams.
+- [Design Tokens Validator](https://animaapp.github.io/design-token-validator-site/) - Validate your design tokens against the Design Token Community Group spec
```
Comment (klaufel) [inline README.md]:
```suggestion
- [Design Tokens Validator](https://animaapp.github.io/design-token-validator-site/) - Validate your design tokens against the Design Token Community Group spec.
```

**Review by klaufel — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (klaufel) [reply README.md]: just add point of line

**Comment by andrico1234** [comment]
Thanks! 🙌🏼

File: README.md (hunk 1 above)
Comment (andrico1234) [reply README.md]: Done 😄

**Pull request merged**